server.go

```diff
@@ -186,6 +235,248 @@ func main() {
+	item := leaderboard_item{}
+	item.UserName = p.GitHubName
+	item.Slug = p.GitHubName
+	item.Score = 0
+	item.Archived = false
+	item.Draft = false
+
+	payload := leaderboard_payload{}
+	payload.Fields = item
+
+	body, err := json.Marshal(payload)
+	if err != nil {
+		return
+	}
+
+	req, err := http.NewRequest("POST", fmt.Sprintf("%s/collections/%s/items?live=true", webflowBaseAPI, webflowCollection), bytes.NewReader(body))
+	if err != nil {
+		return
+	}
+	req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", webflowToken))
+	req.Header.Add("Content-Type", "application/json")
+	req.Header.Add("accept-version", "1.0.0")
+
+	res, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return
+	}
+
+	if res.StatusCode < 200 || res.StatusCode >= 300 {
+		c.Logger().Debug(req)
+		var responseBody []byte
+		_, _ = res.Body.Read(responseBody)
+		c.Logger().Debug(responseBody)
+		// return a real error to the caller to indicate failure
+		errMsg := fmt.Sprintf(msgUpstreamParticipantCreateError, res.Status)
+		if errContext := c.String(http.StatusInternalServerError, errMsg); errContext != nil {
+			err = errContext
+		} else {
+			err = fmt.Errorf(errMsg)
+		}
+		return
+	}
+
+	var response leaderboard_response
+	err = json.NewDecoder(res.Body).Decode(&response)
+	if err != nil {
+		return
+	}
+	id = response.Id
+	return
+}
+
+const msgUpstreamParticipantUpdateError = "could not update score. response status: %s"
+
+func upstreamUpdateScore(c echo.Context, webflowId string, score int) (err error) {
+
+	var payload struct {
+		Fields struct {
+			Score int `json:"score"`
+		} `json:"fields"`
+	}
+	payload.Fields.Score = score
+
+	body, err := json.Marshal(payload)
+	if err != nil {
+		return
+	}
+
+	url := fmt.Sprintf("%s/collections/%s/items/%s?live=true", webflowBaseAPI, webflowCollection, webflowId)
+	req, err := http.NewRequest("PATCH", url, bytes.NewReader(body))
+	if err != nil {
+		return
+	}
+	req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", webflowToken))
+	req.Header.Add("Content-Type", "application/json")
+	req.Header.Add("accept-version", "1.0.0")
+
+	res, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return
+	} else if res.StatusCode < 200 || res.StatusCode >= 300 {
+		c.Logger().Debug(req)
+		var responseBody []byte
+		_, _ = res.Body.Read(responseBody)
+		c.Logger().Debug(responseBody)
+		// return a real error to the caller to indicate failure
+		errMsg := fmt.Sprintf(msgUpstreamParticipantUpdateError, res.Status)
+		if errContext := c.String(http.StatusInternalServerError, errMsg); errContext != nil {
+			err = errContext
+		} else {
+			err = fmt.Errorf(errMsg)
+		}
+	}
+
+	return
+}
+
+var PARTICIPATING_ORGS = map[string]bool{
+	"thanos-io":          true,
+	"serverlessworkflow": true,
+	"chaos-mesh":         true,
+	"cri-o":              true,
+	"openebs":            true,
+	"buildpacks":         true,
+	"schemahero":         true,
+}
+
+func validScore(owner string, name string, user string) bool {
```

Comment:
`name` param is not used

```suggestion
func validScore(owner string, user string) bool {
```

Reply:
done e91d0d2